###  Google Drive マウント

In [ ]:
## Google Drive マウント
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


### OpenAI API呼び出し方法

In [ ]:
!pip install -U openai
#  -Uでは最新のapiが入る。
# !pip install openai==1.3.5

## !pip install openai==0.28

   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 362.9/362.9 kB 7.8 MB/s eta 0:00:00
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 75.6/75.6 kB 4.1 MB/s eta 0:00:00
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 77.9/77.9 kB 4.6 MB/s eta 0:00:00
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 318.9/318.9 kB 16.3 MB/s eta 0:00:00
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.3/58.3 kB 3.1 MB/s eta 0:00:00


### インストール出来たか確認

In [ ]:
import openai
openai.__version__

'1.42.0'

In [ ]:
import os

with open("/content/drive/My Drive/Udemy/7Step_Multi/make_image_interview/key.txt", "r") as f:

# with open("key.txt", "r") as f:
  os.environ['OPENAI_API_KEY'] = f.read()

In [ ]:
## API キー確認
# %env OPENAI_API_KEY


In [ ]:
# 必要なパッケージをインポート
import os
import openai
import pandas as pd
from IPython.display import display, Markdown

In [ ]:
# 環境変数からOpenAI APIキーを設定
# openai.api_key = os.environ["OPENAI"]
openai.api_key = os.environ["OPENAI_API_KEY"]

In [ ]:
import os
print(os.getcwd())

/content


In [ ]:
from openai import OpenAI

client = OpenAI()

audio_file= open('/content/drive/My Drive/AI@WG/Cycle10/data/test_64k.mp3', "rb")

#transcriptions、つまり文字起こしを指定。modelはwhisper-1のみ選択可能。

transcript = client.audio.transcriptions.create(model="whisper-1", file=audio_file, response_format="text")

print(transcript)

今晩は7時になりましたNHK今日のニュースです 大気の状態が非常に不安定になり関東では局地的に1時間に80ミリ以上の猛烈な 雨が降っています 各地で雷雨になっています 夕方 宇都宮市内では 強い雨がたどきつけて視界が悪い状況です 午後5時半までの1時間には宇都宮市で98.5ミリの猛烈な雨が降りました 宇都宮市内の警察や消防によりますと市内では 住宅などの浸水や道路の冠水の通報が相次いで寄せられているということです 埼玉県入馬市付近ではレーダーによる解析で午後3時半までの1時間におよそ100ミリ の猛烈な雨が降ったとみられ 気象庁は記録的短時間大雨情報を発表しました このほか午後5時半までの1時間に 栃木県高根沢町で53ミリの非常に激しい雨が降ったほか 静岡県藤枝市の高根山で43ミリ 長野県羽住野市穂高で35.5ミリの激しい雨が降りました 大気の不安定な状態は今夜遅くにかけて続くほか明日も東日本を中心に大気の状態が非常に不安定になり 非常に激しい雨が降る恐れがあります 明日夕方までの24時間に降る雨の量はいずれも多いところで関東甲心で120ミリ 東海で100ミリと予想され短い時間でこうした雨量に達する恐れがあります 東京電力パワーグリッドによりますと午後5時半現在 関東の4県と山梨県で合わせて7520個が停電しています 停電が発生しているのは埼玉県で3060個群馬県で1760個 茨城県で1340個栃木県で1100個 山梨県で270個となっています 基礎庁は低い土地の浸水や急な川の増水 氾濫土砂災害に警戒するよう呼びかけています 落雷や竜巻などの激しい突風それに氷にも注意が必要です ネジャーで屋外での活動をする人もいると思いますが天気の急変に注意してください 一方小笠原近海を北上している強い台風10号は来週火曜日に非常に強い勢力となり あまり勢力を弱めないまま西日本から東日本にかけて接近する見込みです 来週火曜日や水曜日は大荒れになるほか台風が近づく前から大雨の恐れがあります 台風への備えを進めるようにしてください 基礎庁の観測によりますと強い台風10号は今日午後3時には小笠原近海を北へ進んでいて 中心気圧は980ヘクトパスカル 中心付近の最大風速は35メートル最大瞬間風速は50メートルとなっています 台風は今後も発達を続け来週火曜日にあたる今月27日には四国沖で最大風

In [ ]:
# 音声データの転写結果を格納した変数
transcript_text = transcript


# 要約を実行するためのプロンプトを作成
prompt = f"次のテキストを要約してください.:\n\n{transcript_text}"


from openai import OpenAI
client = OpenAI()

completion = client.chat.completions.create(
    model="gpt-4o-mini",
  messages=[
    {"role": "system", "content": "あなたは要約を行う優秀なアシスタントです."},
    {"role": "user", "content": prompt}
  ]
)

print(completion.choices[0].message)
print(completion)

ChatCompletionMessage(content='今晩のNHKニュースによると、関東地域では非常に不安定な大気の影響で猛烈な雨が降り、視界が悪化しています。特に宇都宮市では98.5ミリの雨が降り、浸水や道路の冠水が報告されています。気象庁は記録的な短時間大雨情報を発表し、今後も東日本を中心に激しい雨が予想されています。また、関東の4県と山梨県で大規模な停電が発生しており、気象状況に警戒が呼びかけられています。\n\nさらに、小笠原近海では強い台風10号が北上しており、来週火曜日には日本に接近する見込みです。台風による強風や大雨の影響で交通機関の運休が予想されており、早めの備えが重要です。\n\n政治面では、自民党の総裁選挙に向けて立候補者が増加しており、石場茂元幹事長が立候補を表明しました。他の候補者も近く立候補を予定しており、激しい戦いが予想されています。また、福島第一原発の処理水放出に対する中国の対応に対して、岸田総理は科学的根拠に基づかないと反発しています。\n\nスポーツでは、大谷翔平選手が史上6人目の1シーズン40本40盗塁を達成しました。明日の天気も不安定で、引き続き注意が必要です。', refusal=None, role='assistant', function_call=None, tool_calls=None)
ChatCompletion(id='chatcmpl-9zxDtfrwXLjbymwkgj9Y4HRdpl3KF', choices=[Choice(finish_reason='stop', index=0, logprobs=None, message=ChatCompletionMessage(content='今晩のNHKニュースによると、関東地域では非常に不安定な大気の影響で猛烈な雨が降り、視界が悪化しています。特に宇都宮市では98.5ミリの雨が降り、浸水や道路の冠水が報告されています。気象庁は記録的な短時間大雨情報を発表し、今後も東日本を中心に激しい雨が予想されています。また、関東の4県と山梨県で大規模な停電が発生しており、気象状況に警戒が呼びかけられています。\n\nさらに、小笠原近海では強い台風10号が北上しており、来週火曜日には日本に接近する見込みです。台風による強風や大雨の影響で交通機関の運休が予想さ

In [ ]:
# 音声データの転写結果を格納した変数
transcript_text = transcript


# 要約を実行するためのプロンプトを作成
prompt = f"次のテキストについて以下の作業を行ってください。.:\n\n{transcript_text},## 打ち合わせのメモをまとめてください。スポーツの情報も拾ってください。##メモから聞いている人が何をしないといけないかをpick upしてください。Action itemが有れば、項目と内容を記載してください。"


from openai import OpenAI
client = OpenAI()

completion = client.chat.completions.create(
    model="gpt-4o-mini",
  messages=[
    {"role": "system", "content": "あなたは優秀なPMOです"},
    {"role": "user", "content": prompt}
  ]
)

print(completion.choices[0].message)
print(completion)

ChatCompletionMessage(content='## 打ち合わせのメモ\n\n### 天気情報\n- **気象状況**: 現在、関東地方では局地的に非常に激しい雨が降っており、雷雨の発生がある。宇都宮市では98.5mmの雨が記録され、浸水や道路冠水の通報が相次いでいる。\n- **今夜の予想**: 大気の不安定な状態は今夜遅くにかけて続く。明日も東日本での非常に激しい雨が懸念されている。\n- **台風情報**: 現在、台風10号が北上中で、来週火曜日には非常に強い勢力になる見込み。特に西日本から東日本では大荒れの天気になる可能性がある。\n\n### 停電情報\n- **停電状況**: 関東の4県と山梨県で7520件の停電が発生中。埼玉県3060件、群馬県1760件など。\n\n### 注意喚起\n- **警戒するポイント**: 低地の浸水、急な川の増水、土砂災害、落雷、竜巻などに注意が必要。\n- **対策**: ゴミの除去、外に置いた物の移動、懐中電灯やカセットコンロの準備、水納の準備など。\n\n### 政治情報\n- **自民党総裁選**: 石場元幹事長が立候補を表明。各候補者は推薦人の確保に奔走中。\n- **立憲民主党代表選挙**: 枝野前代表が観客と握手を交わす様子が報じられる。\n\n### スポーツ情報\n- **大リーグ**: 大谷翔平選手が史上6人目となる1シーズンでのホームラン40本・盗塁40回を達成。\n- **プレー級**: 巨人対中日、巨人が6-2で勝利。日本ハム対ソフトバンクも日本ハムが勝利。\n\n## Action Items\n1. **気象準備**:\n   - 低地に住む人は浸水に備えるために防水対策を進める。\n   - 外に置いてある物（傘、自転車など）を室内に移動する。\n\n2. **停電対策**:\n   - 停電に備えて懐中電灯、カセットコンロ、水を用意しておく。\n\n3. **台風準備**:\n   - 台風10号接近に備え、電池や食料を十分に準備する。\n\n4. **情報の確認**:\n   - 天候や交通機関の運行状況をこまめに確認する。特に新幹線の運行状況に注意。\n\n5. **政治・選挙関係**:\n   - 立候補についての情報を整理し、関係者への確認を進める。', refusa

## 音声データのtext化　(speech to text )
## " 08月25日 午前７時のNHKニュース"のmp3データをtext化
## 9行目のcode実行結果


今晩は7時になりましたNHK今日のニュースです 大気の状態が非常に不安定になり関東では局地的に1時間に80ミリ以上の猛烈な 雨が降っています 各地で雷雨になっています 夕方 宇都宮市内では 強い雨がたどきつけて視界が悪い状況です 午後5時半までの1時間には宇都宮市で98.5ミリの猛烈な雨が降りました 宇都宮市内の警察や消防によりますと市内では 住宅などの浸水や道路の冠水の通報が相次いで寄せられているということです 埼玉県入馬市付近ではレーダーによる解析で午後3時半までの1時間におよそ100ミリ の猛烈な雨が降ったとみられ 気象庁は記録的短時間大雨情報を発表しました このほか午後5時半までの1時間に 栃木県高根沢町で53ミリの非常に激しい雨が降ったほか 静岡県藤枝市の高根山で43ミリ 長野県羽住野市穂高で35.5ミリの激しい雨が降りました 大気の不安定な状態は今夜遅くにかけて続くほか明日も東日本を中心に大気の状態が非常に不安定になり 非常に激しい雨が降る恐れがあります 明日夕方までの24時間に降る雨の量はいずれも多いところで関東甲心で120ミリ 東海で100ミリと予想され短い時間でこうした雨量に達する恐れがあります 東京電力パワーグリッドによりますと午後5時半現在 関東の4県と山梨県で合わせて7520個が停電しています 停電が発生しているのは埼玉県で3060個群馬県で1760個 茨城県で1340個栃木県で1100個 山梨県で270個となっています 基礎庁は低い土地の浸水や急な川の増水 氾濫土砂災害に警戒するよう呼びかけています 落雷や竜巻などの激しい突風それに氷にも注意が必要です ネジャーで屋外での活動をする人もいると思いますが天気の急変に注意してください 一方小笠原近海を北上している強い台風10号は来週火曜日に非常に強い勢力となり あまり勢力を弱めないまま西日本から東日本にかけて接近する見込みです 来週火曜日や水曜日は大荒れになるほか台風が近づく前から大雨の恐れがあります 台風への備えを進めるようにしてください 基礎庁の観測によりますと強い台風10号は今日午後3時には小笠原近海を北へ進んでいて 中心気圧は980ヘクトパスカル 中心付近の最大風速は35メートル最大瞬間風速は50メートルとなっています 台風は今後も発達を続け来週火曜日にあたる今月27日には四国沖で最大風速 45メートル最大瞬間風速は60メートルの非常に強い勢力にまで強まると予想されています その後あまり勢力を弱めないまま西日本から東日本に接近する恐れがあります 台風の進路予想にはまだ幅がありますが西日本から東日本では来週火曜日や水曜日は 雨や風が強まり大荒れの天気となる可能性があります また北日本でも来週水曜日や木曜日は大荒れの天気となる恐れがあります 今回は接近前から大雨となりその後東日本や西日本を台風が直撃する恐れが あります 交通機関も計画運休する可能性があり今後の情報に注意してください JR東日本は台風15の接近に伴い今月27日から28日にかけて 上越新幹線と北陸新幹線を中心とした各新幹線の前線または一部の区間で計画運休を 含む運休のほか大幅な遅れが生じる可能性があると発表しました 台風が接近して雨や風が強まる前に早めに対策を済ませることが大切です 自宅の周りの速攻や排水口などはゴミが詰まると水があふれやすくなるので掃除して おきましょう 暴風で飛ばされたものは住宅や人に被害を与える凶器となります 傘や物干し座を自転車など外に置いてあるものは室内にしまってください 停電が長時間にわたって発生することもあります 懐中電灯やランタンなどのほかにカセットコンロとかスカンなども用意しておきましょう 大雨が降ると集まった雨水が下水道に入りきれず 家の玄関や駐車場が浸水する恐れがあります 土納屋ゴミ袋に水を入れた水納を準備すると安心です ん 来月12日に告示される自民党の総裁選挙です 今日石場元幹事長が立候補することを表明しました 立候補表明は2人目です 石場元幹事長が立候補表明の場に選んだのは地元鳥取県八津町の神社でした 総裁選挙には5回目の挑戦となります 自由民主党総裁選挙に立候補いたします最後の戦いとして 原点に戻りご指示を求めてまいります 石場茂氏は衆議院鳥取区選出の当選12回で67歳 防衛大臣や党の幹事長地方創生担当大臣などを歴任しました ルールを守り 国民に信頼される政治 日本を守り そして国民を守り 地方を守り そして未来を守る 全身全霊で臨んでまいります 地方の再生に全力を挙げる考えを強調するとともに党の政治と金の問題をめぐっては さらに透明性を深めるための努力を最大限に行うと述べ 政治改革に取り組む考えを示しました 一方すでに立候補を表明した小林隆之氏は今日 野党半島地震の被災地を視察しました 被災しても 自分のふるさとに残って ここを建て直していくんだという強い思いを感じまして 今日感じたことを しっかりと政策という形に 落としていきたい 河野デジタル大臣は地元の神奈川県雛塚市で合宿を行う 大相撲の力士たちの健康を見学しました 傾向を見学しましたあさって立候補を表明します 世界的に かなり混盟を深める 誘致と言ってもいい時期になりましたので私の経験をしっかり生かして国をリードして いきたい 小泉慎二郎氏は地元の神奈川県横須賀市の神社を訪れ 浴衣姿で祭りに参加しました かっこいい大祭りのかっこいいね かっこいいね 小泉氏は30日に記者会見し立候補表明する予定です 茂木幹事長も立候補の意向を固め表明時期を検討しています 沖縄県で講演し次のように述べました 総理として この日本の課題解決の 先頭で頑張ってほしい こういう 大きな期待の声をいただいております 近々 決断をしたい 立候補に意欲を示している上川外務大臣は共同議長を務める ティカットアフリカ開発会議の閣僚会合に出席しました 推薦人確保の状況について次のように述べました いよいよまあ着地の姿が見えてきたかなってこんなあの印象であります 改革のチームに入っていただくとこういう思いで今動いていると 野田聖子氏は推薦人を確保に向け働きかけを続けています 出馬の意欲は毎日です女性が幸せになることがこの国を豊かにすることだという メッセージをしっかりと形に残していくことをやっぱりやるための総裁選なんだ 林官房長官は来週にも立候補を表明したい考えで台風10号の状況を踏まえ 判断することにしています 高市経済安全保障担当大臣は近い議員が推薦人の確保にめどがついたとしていて表明する時期を 探っています 自民党総裁選挙の候補者は推薦人が必要となった1972年以降 5人が最も多く今回はそれを上回る過去最多となる可能性を指摘する声も出るなど 今戦の様相が強まっています 一方 立憲民主党の代表選挙です 来月7日の告示まで2週間となりました 立候補正式に表明した枝野前代表は地元の埼玉市で行われた祭りに参加しました 祭りに来た人と次々に握手を交わし頑張ってくださいと声をかけられていました 立候補する意向を固めている 泉代表は地元の京都府内の米国店や田んぼを回って説明を受け 各地のスーパーで米が品薄になっていることを受けて政府に対応を求めていく考えを示し ました 代表選挙に向けては 自分なりにはこの難しい3年間を 努めを果たしてきたという自負はありますから それを皆様にまたどうご評価いただくかということは真摯に受け止めてですね やっていきたいですね また立候補に必要な20人の推薦人の確保について 各議員も真剣に考えて推薦人になるか判断するのでギリギリまで考え続ける人も大勢いると 述べました さらに立候補する方向で調整に入った野田元総理大臣は 栃木県内で代表選挙の在り方について次のように述べました 自民党の総裁選挙の埋没しないように政治と金融問題で これを一番アピールしてほしい いろんな人が手を挙げて推薦に確保できることは望ましいと思いますけれども もはや演出よりポリス度段階だと思います 次です 東京電力福島第一原子力発電所に溜まる処理水の海への放出が始まってから 今日で1年となります 中国は福島第一原発に溜まる処理水の海への放出が始まったことに強く反発して 日本産水産物の輸入を全面的に停止する措置を取り 日本政府は科学的根拠に基づいていないとして即時撤廃を求めています こうした中岸田総理大臣は今日福島県いわき市の魚市場を視察し 中国による日本産水産物の輸入停止措置について次のように述べました 全く科学的根拠に基づかない措置であり 受け入れると受け入れることはできません ビジマとなって引き続きあらゆるレベルで働きかけを行っていきたい 岸田総理大臣は中国による輸入停止措置の影響を踏まえ 今後の漁業支援策の方向性を来週示す意向を明らかにしました ん 大リーグドジャースの大谷翔平選手が史上6人目となる 1シーズンホームライン40本40頭類を達成しました ドジャースは23日本拠地ロサンゼルスでレースと対戦し 大谷選手は一番指名出しで先発出場しました 4回ナイアンダで失礼した後に類へ投類を成功させて大リーグ7年目で自身初となる シーズン40頭類に到達しました そして同点のまま迎えた9回2アウト満塁のチャンスで第5打席 さよなら満塁ホームラン 史上6人目となるホームライン40本40頭類に史上最速の出場126試合目で到達しました 大谷選手はこの勢いがシーズン終了まで続けば ホームラン投類ともに50の大台に乗るペースです シーズンでのホームラン50本50頭類を達成すれば大リーグ史上初めてでこれまでホームラン 45本45頭類を達成した選手もいません また大谷選手がさよならホームランを打つのは大リーグ7年目で初めてで印象的な回帰を達成と なりました ではプレー級デイゲーム3試合の結果です セリーグ巨人対中日は巨人が6対2で勝ちました パリーグ日本ハム対ソフトバンクは日本ハムが6対1で オリックス対ロッテはロッテが1対0で勝ちましたここまでスポーツでした では気象情報です気象予報室の伊藤美雪さんとお伝えしますはい 伊藤さん今日は昨日より広い範囲で赤ラウンが発達していますねはい 列島は元台風7号だった低気圧そして沖縄の南の熱帯低気圧に挟まれる形で 昨日より広い範囲で赤ラウンが湧き立ちやすい気象状況になっています まず昨日北海道の北を通過した元台風7号の低気圧から伸びる寒冷前線が北 日本を通過したこと それと沖縄の南の熱帯低気圧が湿った空気の流れ込みを強めていることで特に九州 から関東付近 大気の状態が非常に不安定になっています 赤ラウンがあまり動かず同じようなところにとどまり 局地的に雨量が多くなってあっという間に寒水や浸水が発生するところもあれば ほとんど降っていないところもある 極端な降り方になっています明日もこの状況が続く恐れがあります 例えば東京の明日の天気はところにより雨雷を伴い非常に激しく降るとあるんですが 大気の状態が不安定なときに単に激しくではなく非常に激しくと 前日から非常にがついているのは珍しいです それだけ明日も交通機関に影響が出るような雷雨に注意が必要だと危機感を持っています そして台風10号の進路も心配ですよね強い台風10号は午後6時には小笠原 近海にあって時速25キロで北上中です 小笠原諸島は明日は波の高さ4メートルと海は湿気 南東の風が強まるでしょう 今のところ27日火曜日の午後非常に強い勢力で四国沖に進み 予想されるコースの中心を進むと28日水曜日に 紀伊半島付近から野戸半島付近を通る可能性があります まだ予報円の範囲は広く西は九州大墨半島 東は東京湾付近まで含まれているので最新情報をご確認ください 去年は台風7号が8月半ばに近畿地方を北上し鳥取県に大雨特別警報が出ました 今回は非常に強い勢力で陸地に近づく予想で台風本体が近づく前から大雨になる 恐れもあり大雨暴風ともに要警戒です 明日以降暑さと雷雨に気をつけつつ台風への備えも進めておきましょう 沖縄は熱帯低気圧の影響で所々で雨や雷雨 九州中国四国近畿東海北陸関東甲信越東北地方は早いところは午前中から広い範囲は 5 所々で雨や雷雨特に東海関東甲信福島では非常に激しく降る恐れがあります 1時間に50ミリ以上に投資が悪くなったり車の運転が危険という振り方です 北海道は曇り時々晴れでしょう 主な都市の明日の予想気温最低最高の順にお伝えします なは27度32度鹿児島27度35度福岡28度34度 高知26度34度広島27度34度 松江26度31度大阪と名古屋は27度34度金沢27度32度新潟26度31度 長野24度30度東京27度34度 仙台25度30度札幌20度28度 くしろ18度23度の予想です明日も大気の状態が不安定というような天気の中で 九州北部から本州の広い範囲が降水確率50%以上となっています 大気の状態が不安定で降水確率50%以上というのは非常に雨が起こりやすい気象 条件となっています 明日も冷や激しい雷雨にご注意ください そして台風15への備えも早めにお願いします また蒸し暑くなります熱中症にも引き続き注意が必要です 皆さんしっかりと準備をなさってください今後の情報にもご注意ください 基礎情報は伊藤美雪さんでした nhk 今日のニュース谷口新一郎でした 7時20分です

## メモまとめ
## 17行目のcode実行結果
### print(completion.choices[0].message)
####  promptをまだまだ、最適化する必要が有ります。

## 打ち合わせのメモ\n\n
### 天気情報\n-
**気象状況**:
現在、関東地方では局地的に非常に激しい雨が降っており、雷雨の発生がある。宇都宮市では98.5mmの雨が記録され、浸水や道路冠水の通報が相次いでいる。\n
- **今夜の予想**:
大気の不安定な状態は今夜遅くにかけて続く。明日も東日本での非常に激しい雨が懸念されている。\n
- **台風情報**: 現在、台風10号が北上中で、来週火曜日には非常に強い勢力になる見込み。特に西日本から東日本では大荒れの天気になる可能性がある。\n\n
### 停電情報\n
- **停電状況**: 関東の4県と山梨県で7520件の停電が発生中。埼玉県3060件、群馬県1760件など。\n\n
### 注意喚起\n
- **警戒するポイント**: 低地の浸水、急な川の増水、土砂災害、落雷、竜巻などに注意が必要。\n
- **対策**: ゴミの除去、外に置いた物の移動、懐中電灯やカセットコンロの準備、水納の準備など。\n\n
### 政治情報\n
- **自民党総裁選**: 石場元幹事長が立候補を表明。各候補者は推薦人の確保に奔走中。\n
- **立憲民主党代表選挙**: 枝野前代表が観客と握手を交わす様子が報じられる。\n\n
### スポーツ情報\n
- **大リーグ**: 大谷翔平選手が史上6人目となる1シーズンでのホームラン40本・盗塁40回を達成。\n
- **プレー級**: 巨人対中日、巨人が6-2で勝利。日本ハム対ソフトバンクも日本ハムが勝利。\n\n
## Action Items\n
1. **気象準備**:\n
   - 低地に住む人は浸水に備えるために防水対策を進める。\n
   - 外に置いてある物（傘、自転車など）を室内に移動する。\n\n
2. **停電対策**:\n
   - 停電に備えて懐中電灯、カセットコンロ、水を用意しておく。\n\n
3. **台風準備**:\n
   - 台風10号接近に備え、電池や食料を十分に準備する。\n\n
4. **情報の確認**:\n
   - 天候や交通機関の運行状況をこまめに確認する。特に新幹線の運行状況に注意。\n\n
5. **政治・選挙関係**:\n
   - 立候補についての情報を整理し、関係者への確認を進める。
